src/components/PineTypes/KeyValuePair.js

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react'
+import * as RegexParser from 'regex-parser'
+import * as isArray from 'lodash/isArray'
+import * as assign from 'lodash/assign'
+import * as some from 'lodash/some'
+import * as includes from 'lodash/includes'
+import * as values from 'lodash/values'
+import * as keys from 'lodash/keys'
+import Input from '../Input'
+import { Flex } from '../Grid'
+
+const normalizeToCollection = value => (isArray(value) ? value : [value])
+
+export const rules = {
+  is: (target, value) => {
+    return some(normalizeToCollection(target), value)
+  },
+  'is not': (target, value) => {
+    return !some(normalizeToCollection(target), value)
+  },
+  'key is': (target, value) => {
```

Comment:
Wouldn't it be great if this could be an object like:
```
'key is': {
  getLabel: (type) => { ... }, // this would be optional
  test: (target, value) => { ... },
}
```
So the operator could also say "tag name" just like the placeholder's input.
This way the SchemaSieve `getOperators()` method would check whether the prop is an object (or not) and whether a getLabel() method and use that to populate the array.

Maybe in the next iteration.

Reply:
I'll implement this suggestion in this PR as it will definitely make the feature more complete